hr/overview/employee_form.rst

```diff
@@ -0,0 +1,123 @@
+
+Complete employee form
+======================
+
+The employee form allows you to add tags, a picture and edit all the general information about your employee.
+You can also create and edit departments and a manager for your employee.
+
+
+.. image:: media/employee-form.png
+   :align: center
+   :alt: Empty employee form
+
+Print
+=====
+
+From the employee form, *Print Badge*, *Time Off Summaries* and *Individual Accounts* for each employee.
+
+.. image:: media/print-actions.png
+   :align: center
+   :alt: Print actions available on the employee form
+
+Action
+======
+
+You can also directly *Archive*, *Delete*, *Duplicate* or *Generate tax forms* and *Attestations* from the employee
+form.
+
+.. note::
+
+   281.10 and 281.45 are tax forms for belgian localisations.
+
+.. image:: media/actions.png
+   :align: center
+   :alt: Actions available on the employee form
+
+Employee Skill management
+=========================
+In the *Resumé* tab in the employee form, you can keep track of the experiences and skills of your employees. Create and
+edit skills and experience by clicking on *Edit* →  **Add**.
+You can also simply delete lines by clicking on the bin icon
+
+.. image:: media/create-skills.png
+   :align: center
+   :alt: Create skills for your employees
+
+Add Skills
+----------
+When creating a Skill you can choose the *Skill Type, the Skill level, who will determine your employee progress.
+Directly create and edit different *Skill Types*, *Skills* and *Skill Levels* from this form
+
+
+.. image:: media/add-experience.png
+   :align: center
+   :alt: Filling skill forms for your employees
+
+Add Experience
+--------------
+When adding an experience, you can pick different categories of experience.
+
+.. image:: media/add-XP.png
+   :align: center
+   :alt: Adding experience for your employees
+
+To add any **experience** or **certification** you need to choose a start date, the End date is optional, you also have
+the possibility to add a description for each experience.
+
+.. image:: media/certification.png
+   :align: center
+   :alt: Certification
+
+.. note::
+   Certifications will be displayed with a trophy next to them.
+
+Tabs
+====
+
+Work information
+----------------
+
+Under the *Work Information* tab, you can assign responsibles for the different other app integrations
+
+.. image:: media/personnal-info.png
+   :align: center
+   :alt: Personal info
+
+Organization chart
+------------------
+
+Setting someone as manager in the employee form will generate an organigram , showing manager and subordinates for the
+employee.
+
+.. image:: media/organization-chart.png
+   :align: center
+   :alt: Organigram
+
+Other tabs include
+==================
+
+**Private Information** gathering the private information about employees, **HR Settings** for employees arrangements
+and **Personal Documents**, allowing you to upload files to have instant access to all relevant information regarding
+your employees
```

Comment:
```suggestion
- **Private Information**: gathers private information about employees,  such as... (continue)
- **HR Settings**:  for employees arrangements (?)
- **Personal Documents**: upload files such as ... *
```

* It would be nice to have some extra info here as the size limit and formats accepted (useful info).